main/ejml-core/test/org/ejml/data/TestDMatrixSparseCSC.java

```diff
@@ -45,25 +45,30 @@ public boolean isStructureValid(DMatrixSparse m) {
         return CommonOps_DSCC.checkStructure((DMatrixSparseCSC)m);
     }
 
+    @Test
+    public void constructor_tooLarge() {
+        assertThrows(IllegalArgumentException.class,()->new DMatrixSparseCSC(1_000_000_000,1_000_000_000,4));
+    }
+
```

Comment:
`10^9` is actually smaller than `Integer.MAX_VALUE`. So this would be actually a valid sparse matrices.

Reply:
(see other comments)